bayesian/api_v1.py

```diff
@@ -557,7 +554,7 @@ class StackAnalyses(ResourceWithSchema):
 
     @staticmethod
     def post():
-        decoded = decode_token()
+        decoded = {}
```

Comment:
what will this change do?

Reply:
sorry. this is a mistake. i had to change this to run in my local setup